app/dockerdwrapperwithcompose.c

```diff
@@ -241,6 +248,35 @@ setup_sdcard(void)
   return res == 0;
 }
 
+/**
+ * @brief Attempt to verify ownership for path with timeout.
```

Comment:
Since we don't produce any Doxygen documentation, do we need to write all of these bulky block comments? I would prefer a much leaner comment that only explains the parts that aren't captured by the function name and the parameter names.